### DISCUSSION TITLE
过期：用补帧就会闪退怎么回事

### DISCUSSION THREAD

**Discussion by mohiko** [hooke007/mpv_PlayKit#37] [回收]
所有补帧脚本都不行

**Comment by hooke007** [comment]
系统版本？mpvlog？

**Reply by mohiko** [reply]
> 系统版本？mpvlog？

windows 专业版 dev 22483.10
[mpv-lazy.log](https://github.com/hooke007/MPV_lazy/files/7422839/mpv-lazy.log)
1

**Reply by hooke007** [reply]
加载python时炸了，兼容性？
版本太前了，我连win11release都没装，暂无法复现

**Reply by hooke007** [reply]
[test.zip](https://github.com/hooke007/MPV_lazy/files/7424464/test.zip)
测试一下这种简易脚本会不会闪退

**Reply by mohiko** [reply]
一样会闪退

**Reply by hooke007** [reply]
那就是兼容性的问题了。暂时还没有看到其他在win11dev下测试vs的报错

**Reply by mohiko** [reply]
> 那就是兼容性的问题了。暂时还没有看到其他在win11dev下测试vs的报错

换了你的mpv.net_CM可以用补帧

**Reply by hooke007** [reply]
cm还是用的lazy8月版的组件。
近期这两我都会更新下一版，到时候再看看

**Reply by mohiko** [reply]
> cm还是用的lazy8月版的组件。 近期这两我都会更新下一版，到时候再看看

到时候我再试一试

**Comment by hooke007** [comment]
lazy已更新10月版

**Reply by hooke007** [reply]
CM已更新11月版

**Reply by mohiko** [reply]
> lazy已更新10月版

新版可以了，不闪退了
